### PR TITLE
Marketplace: Updates categories

### PR DESCRIPTION
**Pull request by Rolilink**
#### Changes proposed in this Pull Request

* Combines eCommerce & Business
* Changes Financial to Finance & Payments
* adds "Booking & Scheduling", "Donations" and "Shipping & Delivery" categories

#### Testing instructions

* Go to `/plugins/:siteSlug`.
* Test that added and updated categories are shown in the dropdown and that results are relevant.

#### Screenshot
image

Related to #62910

### CODE REVIEW

File: client/my-sites/plugins/categories/use-categories.tsx
Comment (paulopmt1) [inline client/my-sites/plugins/categories/use-categories.tsx:71]: I have found `booking-system` tag for `Scheduling` and especially for `Booking` that we could add here.

File: client/my-sites/plugins/categories/use-categories.tsx
Comment (paulopmt1) [inline client/my-sites/plugins/categories/use-categories.tsx:94]: `donation-plugin` was common too if you want to add.

File: client/my-sites/plugins/categories/use-categories.tsx
Comment (paulopmt1) [inline client/my-sites/plugins/categories/use-categories.tsx]: I found `business-directory` could be useful too.

**Review by paulopmt1 — APPROVED** [review]
Good job @Rolilink. Your changes LGTM!

I added a couple of minor suggestions for names.

@vinimotaa I noted, that [after the half of the list](https://dotcommarketplacep2.wordpress.com/2022/04/11/marketplace-plugin-categories/), the order of the category's names is alphabetically unordered. Is that intentional?

image

**Comment by Rolilink** [comment]
Waiting on a design decision here on the dropdown options order @vinimotaa @SaxonF , I will be AFK on Monday so @paulopmt1 feel free to add the tags you mentioned and merge this one when we have a decision.

File: client/my-sites/plugins/categories/use-categories.tsx
Comment (lupus2k) [reply client/my-sites/plugins/categories/use-categories.tsx:94]: I also added `donations` to this one

File: client/my-sites/plugins/categories/use-categories.tsx
Comment (lupus2k) [reply client/my-sites/plugins/categories/use-categories.tsx]: I also added `e-commerce` with a dash

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7280080

Thank you @Rolilink for including a screenshot in the description! This is really helpful for our translators.

**Comment by lupus2k** [comment]
> This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7280080
> 
> Hi @Rolilink, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include all of the following strings:
> 
> * `Booking & Scheduling`
> * `Booking`
> * `Ecommerce & Business`
> * `Finance & Payments`
> * `Shipping & Delivery`
> 
> Thank you in advance!

@a8ci18n Done!

**Comment by Rolilink** [comment]
thanks for merging this one @lupus2k !

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7280080).

**Pull request merged**
